assets/model_monitoring/components/src/model_performance_metrics/compute_metrics.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Evaluator."""
+
+import ast
```

Comment:
nit: Could you please sort the namespaces in the order "system" -> "external"(package) -> "internal" and in alphabetical order within each group?